addons/repair/models/repair.py

```diff
@@ -381,16 +380,15 @@ def _create_invoices(self, group=False):
                     raise UserError(_('No account defined for product "%s".') % fee.product_id.name)
 
                 invoice_line_vals = {
-                    'type': 'out_invoice',
                     'name': name,
                     'account_id': account.id,
                     'quantity': fee.product_uom_qty,
                     'tax_ids': [(6, 0, fee.tax_id.ids)],
                     'product_uom_id': fee.product_uom.id,
                     'price_unit': fee.price_unit,
+                    'price_subtotal': fee.product_uom_qty * fee.price_unit,
```

Comment:
`price_subtotal` is a computed readonly field, this line is useless

Reply:
In this case we need it to update the debit/credit of the invoice lines (See L395 and L401)

Reply:
No. `quantity` + `price_unit` is enough

Reply:
Change the line above by `balance = fee.product_uom_qty * fee.price_unit` then

Reply:
same for `amount_currency` ofc